### PR TITLE
add worspace endpoints to config

### PR DESCRIPTION
**Pull request by mccalluc**
- @jpuerto-psc : Can you confirm that these endpoints are correct? Particularly, that there is only a single DEV workspace API instance that should be referenced from DEV/TEST/STAGE
- @yuanzhou : When was has confirmed, can you add the new envvars, and when that is done, approve?

### CODE REVIEW

**Review by jpuerto-psc — APPROVED** [review]
These look good as of right now. We do not have a service deployed on the TEST VM (nor on STAGE). When this is released we will inform you so that you can update the config.

**Comment by yuanzhou** [comment]
Added new config to DEV instance:
```
# Workspaces REST API and Websocket API

WORKSPACES_ENDPOINT = 'https://workspaces-api.dev.hubmapconsortium.org'
WORKSPACES_WS_ENDPOINT = 'https://ws-workspaces-api.dev.hubmapconsortium.org/'
```
@mccalluc I assume the portal-ui handles the trailing slash regardless if it's present in the config.

File: example-app.conf [hunk 1]
```diff
@@ -25,17 +25,23 @@ TYPE_SERVICE_ENDPOINT =  'https://search-api.test.hubmapconsortium.org'
 ENTITY_API_BASE =        'https://entity-api.test.hubmapconsortium.org'
 ASSETS_ENDPOINT =        'https://assets.test.hubmapconsortium.org'
 XMODALITY_ENDPOINT =     'https://cells.api.hubmapconsortium.org'
+WORKSPACES_ENDPOINT =    'https://workspaces-api.dev.hubmapconsortium.org'
+WORKSPACES_WS_ENDPOINT = 'https://ws-workspaces-api.dev.hubmapconsortium.org/'
```
Comment (mccalluc) [inline example-app.conf:29]:
```suggestion
WORKSPACES_WS_ENDPOINT = 'https://ws-workspaces-api.dev.hubmapconsortium.org'
```

File: example-app.conf [hunk 2]
```diff
@@ -25,17 +25,23 @@ TYPE_SERVICE_ENDPOINT =  'https://search-api.test.hubmapconsortium.org'
...
 
 # GATEWAY_ENDPOINT =       'https://gateway.api.hubmapconsortium.org'
 # ELASTICSEARCH_ENDPOINT = 'https://search-api.dev.hubmapconsortium.org'
 # TYPE_SERVICE_ENDPOINT =  'https://search-api.dev.hubmapconsortium.org'
 # ENTITY_API_BASE =        'https://entity-api.dev.hubmapconsortium.org'
 # ASSETS_ENDPOINT =        'https://assets.dev.hubmapconsortium.org'
 # XMODALITY_ENDPOINT =     'https://cells.api.hubmapconsortium.org'
+# WORKSPACES_ENDPOINT =    'https://workspaces-api.dev.hubmapconsortium.org'
+# WORKSPACES_WS_ENDPOINT = 'https://ws-workspaces-api.dev.hubmapconsortium.org/'
```
Comment (mccalluc) [inline example-app.conf:38]:
```suggestion
# WORKSPACES_WS_ENDPOINT = 'https://ws-workspaces-api.dev.hubmapconsortium.org'
```

File: example-app.conf [hunk 3]
```diff
@@ -25,17 +25,23 @@ TYPE_SERVICE_ENDPOINT =  'https://search-api.test.hubmapconsortium.org'
...
 
 # GATEWAY_ENDPOINT =       'https://gateway.api.hubmapconsortium.org'
 # ELASTICSEARCH_ENDPOINT = 'https://search.api.hubmapconsortium.org'
 # TYPE_SERVICE_ENDPOINT =  'https://search.api.hubmapconsortium.org'
 # ENTITY_API_BASE =        'https://entity.api.hubmapconsortium.org'
 # ASSETS_ENDPOINT =        'https://assets.hubmapconsortium.org'
 # XMODALITY_ENDPOINT =     'https://cells.api.hubmapconsortium.org'
+# WORKSPACES_ENDPOINT =    'https://workspaces.api.hubmapconsortium.org/'
+# WORKSPACES_WS_ENDPOINT = 'https://ws-workspaces.api.hubmapconsortium.org/'
```
Comment (mccalluc) [inline example-app.conf:47]:
```suggestion
# WORKSPACES_ENDPOINT =    'https://workspaces.api.hubmapconsortium.org'
# WORKSPACES_WS_ENDPOINT = 'https://ws-workspaces.api.hubmapconsortium.org'
```

**Review by mccalluc — COMMENTED** [review]
Zhou noticed the extra slashes.

**Comment by mccalluc** [comment]
@yuanzhou
- Thank you for noticing the slashes: That was not my intention. They may not hurt, but better not to include them. I've made the corrections.
- The production workspace endpoints are up now, so please configure them as indicated. Every environment should have `WORKSPACES_ENDPOINT` and `WORKSPACES_WS_ENDPOINT` set.

Thanks! I'll re-request review from you. Approve when every environment is configured.

**Review by yuanzhou — APPROVED** [review]

**Pull request merged**